.github/workflows/maven.yml

```diff
@@ -38,8 +38,8 @@ jobs:
         distribution: 'adopt'
         java-version: ${{ matrix.java-version }}
         cache: maven
-    - name: Build and Skip Tests (Maven)
-      run: mvn clean install -DskipTests=true
```

Comment:
for purposes of getting the error logs i have locally, i have removed the skipping of tests during build